packages/app/src/firebaseNamespaceCore.ts

```diff
@@ -182,7 +182,10 @@ export function createFirebaseNamespaceCore(
   ): FirebaseServiceNamespace<FirebaseService> {
     // Cannot re-register a service that already exists
```

Comment:
The comment is outdated and needs update.

Reply:
I think we should log a message at DEBUG log level for catching the possible version mixup. We already use logger in `@firebase/app`, we can make it sharable and use it here.

Reply:
Done.